specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -9968,6 +10849,16 @@
       ],
       "description": "Describes a data disk."
     },
+    "CapacityReservationProfile": {
+      "type": "object",
+      "properties": {
+        "capacityReservationGroup": {
+          "$ref": "#/definitions/SubResource",
+          "description": "Describes the parameter of capacity reservation resource id that can be used for allocating the virtual machine. <br><br> NOTE:Please refer https://aka.ms/CapacityReservation for more details."
```

Comment:
Describes the parameter of capacity reservation group resource id that can be used for allocating the virtual machine. Please refer to https://aka.ms/CapacityReservation for more details.

Reply:
updated